OMNA/bcast/bcast-indentification-authority-registry.html

```diff
@@ -50,7 +50,7 @@
         <li>1.<i> {IAiD}</i> patterns starting with 'x-' are for testing and need not be coordinated with OMNA as they will not be recorded </li>
```

Comment:
This file doesn't display properly. It looks different from the other copied registries.